File: content/fixtures/code.rb
Task = Data.define(:queue, :priority, :detail)

def format_tasks(tasks)
  tasks.map do |task|
    "#{task.queue} priority #{task.priority}: #{task.detail}"
  end
end

tasks = [
  Task.new("imports", 2, "source file received"),
  Task.new("reviews", 1, "extraction fixture marker: shared code sample"),
  Task.new("exports", 3, "summary report queued")
]

puts(format_tasks(tasks))

File: content/fixtures/code.cs
using System;
using System.Collections.Generic;
using System.Linq;

internal static class Code {
    private sealed record ReviewStep(string Name, string Owner, string Detail);

    private static IEnumerable<string> Describe(IEnumerable<ReviewStep> steps) {
        return steps.Select(step => $"{step.Name} owned by {step.Owner}: {step.Detail}");
    }

    private static void Main() {
        var steps = new[] {
            new ReviewStep("Intake", "Elena", "documents received"),
            new ReviewStep("Validation", "Kai", "extraction fixture marker: shared code sample"),
            new ReviewStep("Archive", "Rae", "retention label applied"),
        };

        foreach (var line in Describe(steps)) {
            Console.WriteLine(line);
        }
    }
}
